### DISCUSSION TITLE
GitHub page not launching when forwarded to a Godaddy domain

### DISCUSSION THREAD

**Discussion by shivanisart** [community/community#77923] [New to GitHub]
He’ll. I need a little help. I have a repository shivani-mathur in my account [shivanimathurart@gmail.com](mailto:shivanimathurart@gmail.com).
This repository has been forwarded to a Godaddy domain called shivanimathur.com

I have two problems:

If I type shivanimathur.com in a search bar it launches the website , but if I type [www.shivanimathur.com](http://www.shivanimathur.com/) or http://shivanimathur.com/ it says server not found . I called go daddy and they directed me to the dns records  and ask me to call GitHub .

My second problem is If I access this from shivanimathur.com the font size is different than if I access it from https://shivanisart.github.io/Shivani-Mathur/aboutshivani/aboutshivani.html

**Comment by ereeshika** [comment]
It seems like you have two distinct issues. I'll address Domain not working with www or without http

This issue is likely related to DNS configuration. Make sure that your domain's DNS settings are correctly configured to point to GitHub Pages. Here's what you can do:

GitHub Repository Settings:
Go to your GitHub repository ("shivani-mathur").
Navigate to the "Settings" tab.
Scroll down to the "GitHub Pages" section.

Custom Domain:
In the "Custom domain" section, make sure you have entered your domain name (shivanimathur.com).
If you want to use www as well, add [www.shivanimathur.com](http://www.shivanimathur.com/).

DNS Records:
Go to your GoDaddy account.
Check the DNS settings for your domain.
Add DNS records for both shivanimathur.com and [www.shivanimathur.com](http://www.shivanimathur.com/) that point to GitHub's IP addresses.

Propagation:
DNS changes may take some time to propagate. Be patient, and try accessing [www.shivanimathur.com](http://www.shivanimathur.com/) and http://shivanimathur.com/ after a while.
